angular-workspace/projects/ni/nimble-angular/rich-text-mention/nimble-rich-text-mention-base.directive.ts

```diff
@@ -0,0 +1,42 @@
+import { Directive, ElementRef, EventEmitter, HostListener, Input, Output, Renderer2 } from '@angular/core';
+import type { RichTextMentionValidity } from '@ni/nimble-components/dist/esm/rich-text-mention/base/models/mention-validator';
+import type { RichTextMention } from '@ni/nimble-components/dist/esm/rich-text-mention/base';
+
+export type { RichTextMentionValidity };
```

Comment:
Fully export so clients can access members.
```suggestion
export { RichTextMentionValidity };
```